.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ on:
 
 jobs:
   fmt:
-    runs-on: ubuntu-latest
+    runs-on: macos-14
     steps:
```

Comment:
Any reason that the format job has been changed to MacOS? Unless I'm mistaken, having it be ubuntu _should_ give us _slightly_ better perf, so if not crucial, it would be nice for this to just stay Ubuntu.

Reply:
It turns out `macos-14` runners have three cores instead of two: this may help for `fmt` too, as we do verusfmt in parallel, but I admittedly just changed it to be consistent with the primary test job.

Reply:
Ah, I didn't know the number of cores thing was different, TIL!